.php-cs-fixer.dist.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-declare(strict_types=1);
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__);
 
-$finder = (new PhpCsFixer\Finder())
-    ->in(__DIR__)
-;
-
-return (new Redaxo\PhpCsFixerConfig\Config())
```

Comment:
Gab es `Redaxo\PhpCsFixerConfig\Config()` evtl mal? Ich habe das auch in denn anderen AddOns gesehen. So allerdings lässt sich PHPCS Lokal nicht ausführen:
![image](https://github.com/user-attachments/assets/b95c6e98-9e11-4d25-a935-6f21ba0527da)

Reply:
Gibt es noch: https://github.com/redaxo/php-cs-fixer-config Das kannte ich noch nicht. Ich habe nun das Package eingebunden.

Reply:
Vermutlich ist es deswegen auch bei denn anderen AddOns kaputt..

Reply:
Ich weiß leider einfach nicht, wie es richtig gehört.

Reply:
Dort ist Ja auch beschrieben, wie es zu verwenden ist. Ich kannte das Repository bis dato nicht.

Reply:
Ich denke, so wie du das beschreibst, ist es richtig, aber ich hätte da gerne ein 2. Paar Augen (wenn ich oder Thomas Skerbis es nicht sind)